Homework2/Homework2/SymbolString.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Homework2
+{
+    public class SymbolString
+    {
+       public static int DifferentElem(string str)
+        {
+            int count = 1;                      //counter to register the number of testing elements 
+            int length = str.Length;            //length of the line
+            int maxElem = 0;
+            for (int i = 0; i < length - 1; i++)     //finding max number of different elements
+            {
+                if (str[i] != str[i + 1])
+                {
+                    count++;
+
+                    if (count > maxElem)
+                    {
+                        maxElem = count;
+                    }
+                }
+                else
+                {
+                    count = 1;
+                }
+
+            }
+            return maxElem;
+        }
+
+        public static int EqualNumbers(string str)
+        {
+            int count = 1;                      //counter to register the number of testing elements 
+            int length = str.Length;            //length of the line
+            int maxNumb = 0;
+            for (int i = 0; i < length - 1; i++)     //finding max number of repeating numbers
+            {
+                if (str[i] >= 48 && str[i] <= 57)
+                {
+                    if (str[i] == str[i + 1])
+                    {
+                        count++;
+
+                        if (count > maxNumb)
+                        {
+                            maxNumb = count;
+                        }
+                    }
+                    else
+                    {
+                        count = 1;
+                    }
+                }
+
+            }
+            return maxNumb;
+        }
+
```

Comment:
Think how to combine method EqualNumbers and EqualLatin with common functionality? they do almost the same. I propose to use special method for Check IsLatin or Digit inside